app/src/main/res/values/localized.xml

```diff
@@ -159,7 +159,9 @@
     <string name="p_battle_config_cmd_text_summary">Show text-box for editing Skill commands</string>
 
     <string name="p_wait_ap_regen_text">Wait for AP Regeneration</string>
-    <string name="p_wait_ap_regen_text_summary">Keeps FGA running even when AP runs out, and automatically restart node once enough AP has regenerated.</string>
+    <string name="p_wait_ap_regen_text_summary">Keeps FGA running even when AP runs out and automatically restart node once enough AP has regenerated.</string>
+    <string name="p_wait_ap_regen_minutes_text">Minutes Before Rechecking</string>
+    <string name="p_wait_ap_regen_minutes_text_summary">Default: 5. Please set your screen timeout to be longer than the duration.</string>
```

Comment:
```suggestion
    <string name="p_wait_ap_regen_minutes_text_summary">Please set your screen timeout to be longer than the duration.</string>
```

The default isn't really crucial to know for users unlike fine-tune. So, I think we could omit that from the summary.

Reply:
For the translation too